### ISSUE TITLE
Include ALL innate special rule values in the output, not just Tough

### ISSUE THREAD

**Issue by thomascgray** [thomascgray/opr-af-to-tts#24]
Some models get special rules that come with values other than Tough.

For example, vehicles:
![image](https://github.com/thomascgray/opr-af-to-tts/assets/5067968/95df79c3-c14a-4f0f-ac82-813f6e31fc3b)

We need to include those values in the output somewhere

**Comment by thomascgray** [comment]
This is currently blocked by Army Forge TTS endpoint not returning the rules text for custom special rules.

See https://army-forge.onepagerules.com/api/tts?id=I_PB05NtLD_j for example - "Avenger" is a custom special rule and doesn't have any rule text (unlike something like "Shield Wall" in that example, that you can see does have rule text)

**Comment by thomascgray** [comment]
Solved by https://github.com/thomascgray/opr-af-to-tts/releases/tag/1.6

**Comment by spacenavy90** [comment]
Related issue I noticed, Psychic may not be affected by this change, especially not Psychic upgrades:

![image](https://github.com/thomascgray/opr-af-to-tts/assets/10876259/5b10a294-bab4-4a63-8217-4438661ef3aa)

Notice here that no Psychic (X) is shown and there is no indication on the model stat except the assignment card that it received a Psychic(+1) upgrade.

**Comment by thomascgray** [comment]
Ooh, great spot. I think this is likely something to do with the fact that you took Psychic as an upgrade I'm guessing? And the change is to render out the rating for all special rules that are _innate_ to the model - e.g comes with it as standard.

I'll make a new issue with your details, as I think this is a specific thing that needs to be handled. Thanks for the report1
